packages/plugin/__test__/__snapshots__/test.js.snap

```diff
@@ -1875,6 +1875,24 @@ exports[`typescript ts-class-controller-extension-usage-new.ts 1`] = `
 });"
 `;
 
+exports[`typescript ts-class-controller-extension-wrapped.ts 1`] = `
+"sap.ui.define(["sap/ui/core/mvc/Controller", "sap/ui/core/mvc/ControllerExtension", "sap/fe/core/controllerextensions/Routing"], function (Controller, ControllerExtension, Routing) {
+  "use strict";
+
+  const cov_1uvvg22e7l = () => {
+    return {
+      "s": {}
+    };
+  };
+  const MyExtendedController = Controller.extend("test.controller.MyExtendedController", {
+    routing3: Routing.override({}),
```

Comment:
To have a proper instrumentation I would suggest to keep the first value of the bracket and write it like that:

```
routing: (cov_1uvvg22e7l().s[5]++, Routing),
routing2: (cov_1uvvg22e7l().s[5]++, Routing.override({})),
routing3: (cov_1uvvg22e7l().s[5]++, Routing.override({}))
```

Reply:
true, will do